code/modules/mob/living/carbon/human/human_damage.dm

```diff
@@ -500,3 +500,6 @@ This function restores all organs.
 		if(reagents.has_reagent(LITHOTORCRAZINE))
 			rads = rads/2
 	return ..()
+
+/mob/living/carbon/human/insectoid/apply_radiation(var/rads, var/application = RAD_EXTERNAL)
```

Comment:
Did you test this to make sure it works? I think most players are just default humans with a set species

Reply:
Scroll up like five comments